### PR TITLE
Js post orientation

### PR DESCRIPTION
**Pull request by JoeShep**
In preparation for the post-sprint 2 topics, I have updated all the ORM-related resource/exercise files to strip out Knex/Bookshelf and insert Sequelize. Because of the lack of documentation for migrations and seeding I have been fairly verbose, so there's ample opportunity for confusing or error-filled stuff. This is definitely a first draft. I have not re-read this stuff to check for flow and ease-of-use from one file to the next, which will be important, since it's pretty much one inter-connected exercise. @gregkorte and @Jufebrown your insights are appreciated since you are new to Sequelize.

### CODE REVIEW

File: post-orientation-exercises/postgres/01-postgresql-intro.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # PostgreSQL
 
 PostgreSQL is a powerful, open source object-relational database system. As a database server, its primary function is to store data securely, and to allow for retrieval at the request of other software applications.
-
+PostgreSQL ( or just Postgres, as we lazy devs tend to call it ), is one of a number of SQL dialects. Language-wise, it's indistinguishable from the syntax you have used to this point in SQLite. The big difference for you will the form PostgreSQL will take as a db. No more simple document stored in your project files. Instead you will spin up a dedicated db server. It will still be run locally, so no deployment yet, but it will take some new tools and methods to get used to. Installation is different depending on your platform ( Mac or PC ) and will be covered in class.
```
Comment (JoeShep) [inline post-orientation-exercises/postgres/01-postgresql-intro.md]: found a typo. "The big difference for you will the form "

**Review by Jufebrown — COMMENTED** [review]
moved this comment to the line in question

File: post-orientation-exercises/postgres/05-sequelize-seeding.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+### Seeding
+
+You've seeded data before, but Sequelize is going to make the process a bit easier (although official documentation for this process is severely lacking as of this writing).
+
+Just like migrations, creating seed files with Sequleize requires the CLI tool. And, similar to migrations, you can create a seed file by running `sequelize seed:create --name my-seed-file`
+
+Knex will create a seed file with a timestamp and a `.js` extension. Open it up and it looks like this:
```
Comment (Jufebrown) [inline post-orientation-exercises/postgres/05-sequelize-seeding.md]: Knex should be Sequelize

File: post-orientation-exercises/postgres/02-intro-to-ORMs.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+├── config
+│   └── config.json
+├── migrations
+├── models
+│   └── index.js
+└── package.json
+└── seeders
+```
+
+Open up `congig/config.json` in your editor. Here, you will see three configurations for three different environments: development, testing, and production. That's cool. Now you can mess around with building db in development mode, run tests against a whole other version in test mode, and keep a clean version, free from fake data and db-breaking experiments, when running in production.
```
Comment (Jufebrown) [inline post-orientation-exercises/postgres/02-intro-to-ORMs.md]: should be `config/config.json`

File: post-orientation-exercises/postgres/03-sequelize-models.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
+# Models
+Models are a representation in our code of how our data is structured in our databse.
```
Comment (Jufebrown) [inline post-orientation-exercises/postgres/03-sequelize-models.md]: typo: databse => database

File: post-orientation-exercises/postgres/05-sequelize-seeding.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+To seed your database, use `sequelize db:seed:all` to run all the seed files or `sequelize db:seed --seed seed_file_name` to seed one file. With `all`, the migrations are run by timestamps in ascending order. If you rename the files without the timestamps, you can name them in alphabetical order and they will run from a - z.
+
+When working with foreign key relationships, you will want to make sure that your seed fields are named appropriately so that tables that do not require foreign keys get seeded first, and the tables that rely on them are seeded afterward.
+
+## Exercise
+
+Back to the database we made in the previous exercise. Go ahead and run those migrations so that our tables exist, and let's seed them with data.
+
+1. Create a seed file for your beaches and add at least three beaches to your database.
+1. Create a new seed file for your sandcastle tools and add some of those to the databse.
```
Comment (Jufebrown) [inline post-orientation-exercises/postgres/05-sequelize-seeding.md]: typo: databse -> database

File: post-orientation-exercises/postgres/06-sequelize-relationships.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
+# Sequelize Associations, or how to deal with bridge/associative/pivot/join/thingamajig tables and foriegn key relationships
```
Comment (Jufebrown) [inline post-orientation-exercises/postgres/06-sequelize-relationships.md]: typo: foriegn -> foreign

**Comment by JoeShep** [comment]
I looked at the templating exercises and decided they needed updating. Also need a look, @Jufebrown @gregkorte

File: post-orientation-exercises/express-static/02-templating-pug.md [hunk 7]
```diff
@@ -4,12 +4,57 @@
...
 
 #### Pug
 
 Pug is a clean, whitespace sensitive syntax for writing html. We will be using the Pug templating engine with our Express server. Check out the [PugJs docs](https://pugjs.org/api/getting-started.html) for more information.
 
+BTW, Pug used to be called Jade. You will see references to Jade all over the place on the interwebs, since it was called that for a number of years.
+
+Pug and Handlebars are very similar in their goals and end results. You're still going to bind javascript values to templates to create views to render in a broswer. But the implementation is slightly different. the first thing you'll notice is the syntax. Pug uses HTML tags, but in a whitespace-dependent syntax that does away with angle brackets (`< >`) and closing tags. Proper indentation is _required_. Without properly indented
```
Comment (Jufebrown) [inline post-orientation-exercises/express-static/02-templating-pug.md:15]: typo: broswer -> browser

**Comment by JoeShep** [comment]
Merging to have on hand for class. But will need refactor at some point

File: post-orientation-exercises/express-static/02-templating-pug.md (hunk 7 above)
Comment (gregkorte) [reply post-orientation-exercises/express-static/02-templating-pug.md:15]: typo: 'the first thing' -> 'The first thing'

**Pull request merged**